sdk/src/main/kotlin/org/ocast/sdk/core/ReferenceDevice.kt

```diff
@@ -111,15 +111,18 @@ open class ReferenceDevice(upnpDevice: UpnpDevice) : Device(upnpDevice), WebSock
     /** The web socket. */
     protected var webSocket: WebSocket? = null
```

Comment:
duplicate websocket object, remove `webSocket`, use `webSockets` instead

into `sendToWebSocket` : use `webSockets[0]` (or named map "settings" for example)